src/flows/pipes/Visualization/ExportDashboardOverlay/DashboardDropdown.tsx

```diff
@@ -23,18 +23,20 @@ const DashboardDropdown: FC = () => {
   if (dashboards.length) {
     menuItems = (
       <>
-        {dashboards.map((dashboard, i) => (
-          <Dropdown.Item
-            key={`${dashboard.name}${i}`}
-            value={dashboard}
-            onClick={dashboard => handleSetDashboard(dashboard)}
-            selected={dashboard.name === selectedDashboard?.name}
-            title={dashboard.name}
-            wrapText={true}
-          >
-            {dashboard.name}
-          </Dropdown.Item>
-        ))}
+        {dashboards
+          .sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
does [this line](https://github.com/influxdata/ui/pull/1916/files#diff-53c7cf5df4c58cb003803675516b1a318993ee1c4b9bbdd94addc32e9271d940R11) above which pulls them from the selector not sort them? it looks like it's doing the same comparison

Reply:
lol yup. After opening this PR I was like "maybe I should make sure this is still an issue". It isn't, it's already been fixed 🤦